### PR TITLE
Change move_wheel order into movement_order

### PR DESCRIPTION
**Pull request by bonnetn**
Move wheels has been deprecated. 

We now must send a (type: uint8, ticks: int32) packet to the LL to translate or rotate.

### CODE REVIEW

File: src/robot/controller/localization_test.py [hunk 1]
```diff
@@ -94,10 +95,9 @@ def test_odometry_position(localization_controller,
         """
         Test that move forward calls the gateway.
         """
-        asyncio.create_task(localization_controller.move_forward(10))
+        asyncio.create_task(localization_controller.move_forward(2 * math.pi))
```
Comment (tictacmenthe) [inline src/robot/controller/localization_test.py]: what?
Shouldn't it be rotate? since you're using PI

File: src/robot/gateway/motion_test.py [hunk 2]
```diff
@@ -13,11 +13,22 @@ class TestMotionGateway:
...
+    async def test_rotate(can_adapter_mock):
         """
         Test that the motion gateway calls the CAN adapter.
         """
         motion_gateway = MotionGateway(can_adapter=can_adapter_mock)
-        await motion_gateway.move_wheels(1, 2)
-        can_adapter_mock.send.assert_called_once_with(
-            0x00000000010, b'\x01\x00\x00\x00\x02\x00\x00\x00')
+        await motion_gateway.rotate(15)
+        can_adapter_mock.send.assert_called_once_with(0x00000000010,
```
Comment (tictacmenthe) [inline src/robot/gateway/motion_test.py]: correct IDs

File: src/robot/gateway/motion_test.py [hunk 3]
```diff
@@ -13,11 +13,22 @@ class TestMotionGateway:
...
+
+    @staticmethod
+    @pytest.mark.asyncio
+    async def test_translate(can_adapter_mock):
+        """
+        Test that the motion gateway calls the CAN adapter.
+        """
+        motion_gateway = MotionGateway(can_adapter=can_adapter_mock)
+        await motion_gateway.translate(15)
+        can_adapter_mock.send.assert_called_once_with(0x00000000010,
```
Comment (tictacmenthe) [inline src/robot/gateway/motion_test.py]: correct IDs

File: src/simulation/handler/simulation.py [hunk 4]
```diff
@@ -39,58 +34,44 @@ def __init__(self, configuration: Configuration, event_queue: EventQueue,
...
+        if msg.ticks == 0:
+            self.event_queue.push(EventOrder(type=EventType.MOVEMENT_DONE), 0)
             return
 
-        rot_speed = self.simulation_configuration.rotation_speed
+        ticks_per_revolution = self.configuration.encoder_ticks_per_revolution
+        rotation_speed = self.simulation_configuration.rotation_speed
+        tickrate = self.simulation_configuration.tickrate
+
+        revolution_to_rotate = abs(msg.ticks) / ticks_per_revolution
```
Comment (tictacmenthe) [inline src/simulation/handler/simulation.py]: abs?

File: src/util/can_id.py [hunk 5]
```diff
@@ -3,7 +3,7 @@
 """
 HEARTBEAT = 0x00010101010
 PROPULSION_STOP = 0x00000000000
-PROPULSION_MOVE_WHEELS = 0x00000000010
+PROPULSION_MOVEMENT_ORDER = 0x00000000010
 PROPULSION_MOVEMENT_DONE = 0x00000000001
 PROPULSION_ENCODER_POSITION = 0x00000000011
 SERVOS_SET_ANGLE = 0x00100000000
```
Comment (tictacmenthe) [inline src/util/can_id.py]: wrong IDs, should be 11 bits (pipe id, order id, board id)

File: src/robot/controller/localization_test.py (hunk 1 above)
Comment (bonnetn) [reply src/robot/controller/localization_test.py]: Nah, test configuration is that wheel radius = 1, and that encoders emit 1 tick / rotation.

This means, 2pi moves the robot forward by 1 tick.

**Pull request merged**